pkg/platform/gitlab.go

```diff
@@ -66,15 +66,15 @@ type gitLabConfig struct {
 	GuardianGitLabToken string
 	GitLabBaseURL       string
 
-	GitLabProjectID      int
-	GitLabMergeRequestID int
+	GitLabProjectID       int
+	GitLabMergeRequestIID int
 }
 
 type gitLabPredefinedConfig struct {
-	CIJobToken       string
-	CIServerHost     string
-	CIProjectID      int
-	CIMergeRequestID int
+	CIJobToken        string
+	CIServerHost      string
+	CIProjectID       int
+	CIMergeRequestIID int
```

Comment:
Can you add a comment similar to the PR body explaining why, doc link, etc

Reply:
Added